Sources/MockoloFramework/Models/ParsedEntity.swift

```diff
@@ -94,11 +94,13 @@ final class Entity {
         return attributes.filter {$0.contains(String.available)}
     }
 
-    static func model(name: String,
-                      label: String = "",
+    static func model(label: String = "",
```

Comment:
this is just a placeholder function to be called from swiftsyntax parser; it will be replaced with real models after #58 is merged.